mobile/src/src/components/Housing/Listings/ListingsContainer.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+import { StyleSheet, View, FlatList } from "react-native";
+import ListingView from "./ListingView"
+import { ListingsEndpoint } from "../../../utils/endpoints";
+
+class ListingContainer extends Component{
+
+    state = {
+        listings: [],
+        extendListings: (newListings) => {
+            this.setState({
+                listings: this.state.listings.concat(newListings),
+            });
+        },
```

Comment:
We probably don't have to make this function part of the state. We can just make this function part of the class.